app/scripts/controllers/dashboard.coffee

```diff
@@ -98,12 +98,25 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', (
         watchCompute 'ptratioComputedMax',
           dependencies: ['allSchools']
           waitForPromise: true
-          computer: ([allSchools]) -> utils.max allSchools, 'PUPIL_TEACHER_RATIO', 10
+          computer: ([allSchools]) ->
+            max = utils.max allSchools, 'PUPIL_TEACHER_RATIO', 100
+            if max?
+              max.then (value) -> if value != $scope.range.ptratio.max
+                $scope.range.ptratio.max = value
```

Comment:
:cry: oh well

not lots more features to add, so I'll just merge it. and then we can get to the fun stuff in React!!!!